pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -605,6 +653,89 @@ func ExecuteLedCmd(deviceNetworkStatus *types.DeviceNetworkStatus,
 	}
 }
 
+// Execute blink forever until there is a message to stop, do we really need blinkcount here ?
+// Just blink until blinkcount before checking for exit stop cmd.
+func ExecuteBlinkLoop(color string, blinkCh chan string) {
+
+	log.Functionf("Started blink thread for color %s", color)
+
+	var file, err = os.Create(blinkInProgress)
+	if err != nil {
+		return
+	}
+	file.Close()
+
+	for fileExists(blinkInProgress) {
```

Comment:
The golang ideom to do this is to have the caller create a channel (e.g., called doneChan := chan strunct{}), have this function conditionally receive from that channel each time around the loop, and have the caller send the empty struct when wants the blinking to stop.

Reply:
Will use two channels as discussed one to request stop and other to receive the ack.